app/views/posts/show.html.erb

```diff
@@ -17,8 +17,13 @@
     <div class="post-paragraph">
         <p><%= @post.text %></p>
     </div>
+    <div class="delete-button-container">
+     <% if @post && can?(:destroy, @post) %>
+        <%= button_to "Delete Post", user_post_path(user_id: @user.id, id: @post.id), method: :delete, class: "delete-btn" %>
+      <% end %>
+    </div>
     <%= form_with model: @like, url: user_post_likes_path(post_id: @post.id) do |f| %>
-      <span><%= f.submit "Like", class: "btn" %></span>
+      <span><%= f.submit "Like", class: "btn-delete" %></span>
     <% end %>
   </div>
 

```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM  thanks for the advice! I will keep it in mind, I already added the button to delete comments too, I thought I added it but it was just the post button... thank you!